package.json

```diff
@@ -2,9 +2,10 @@
   "scripts": {
     "clean": "rm -rf node_modules yarn-error.log css static/css *.log *.sqlite _site/ build/ .jekyll-metadata .bundle",
     "watch": "watch -p 'static/sass/**/*.scss' -c 'yarn run build'",
-    "build": "yarn run build-css && yarn run build-js",
+    "build": "yarn run build-css && yarn run build-js && yarn run build-latest-news",
     "build-css": "sass static/sass/styles.scss static/css/styles.css --load-path=node_modules --style=compressed && postcss --map false --use autoprefixer --replace 'static/css/**/*.css'",
     "build-js": "mkdir -p static/js/modules/cookie-policy && cp node_modules/@canonical/cookie-policy/build/js/cookie-policy.js static/js/modules/cookie-policy",
+    "build-latest-news": "mkdir -p static/js/modules/latest-news && cp node_modules/@canonical/latest-news/dist/latest-news.js static/js/modules/latest-news",
```

Comment:
It doesn't work on demo. It seems that in the [Dockerfile](https://github.com/canonical-web-and-design/canonical.com/blob/main/Dockerfile) we are using the `build-js` step as the single step to build all JS, so you might want to refactor this in a similar fashion as [u.com package.json](https://github.com/canonical-web-and-design/ubuntu.com/blob/main/package.json) so that it uses build-js